src/hooks/screen/useSignedChatInfoHook.ts

```diff
@@ -2,37 +2,38 @@ import * as React from 'react';
 import {useNavigation} from '@react-navigation/core';
 
 import ChannelList from '../../database/schema/ChannelListSchema';
-import UseAnonymousChatInfoScreenHook from '../../../types/hooks/screens/useAnonymousChatInfoScreenHook.types';
+import UseSignedChatInfoScreenHook from '../../../types/hooks/screens/useSignedChatInfoScreenHook.types';
 import useChatUtilsHook from '../core/chat/useChatUtilsHook';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
 import {ChannelListMemberSchema} from '../../../types/database/schema/ChannelList.types';
 import {getAnonymousUserId} from '../../utils/users';
 import {getUserId} from '../../utils/token';
 
-function useAnonymousChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
+function useSignedChatInfoScreenHook(): UseSignedChatInfoScreenHook {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function name has been changed from `useAnonymousChatInfoScreenHook` to `useSignedChatInfoScreenHook`. Make sure all references to this hook are updated accordingly.



<!-- This is an auto-generated comment by OSS CodeRabbit -->